CRM/Upgrade/Incremental/php/FiveSixtyNine.php

```diff
@@ -37,6 +43,16 @@ public function upgrade_5_69_alpha1($rev): void {
     $this->addTask('fix crmDate for installs that existed pre-5.43 - end date',
       'updatePrintLabelToken', 'event.end_date|crmDate:"%B %E%f}"', 'event.end_date|crmDate:\\\"%B %E%f\\\"}"', $rev
     );
+    $this->addTask('Make civicrm_membership.title required', 'alterColumn', 'civicrm_membership_type', 'title',
+      "varchar(255) COLLATE utf8mb4_unicode_ci NULL COMMENT 'Membership Type title. For top of page display"
+    );
+    $this->addTask('Add MembershipType.frontend_title', 'alterColumn', 'civicrm_membership_type', 'frontend_title',
+      "varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'Membership Type title'"
+    );
+    $this->addTask('Add MembershipType.frontend_title', 'addColumn', 'civicrm_membership_type', 'frontend_title',
+      "varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'Membership Type Public title'"
+    );
+    $this->addTask(ts('Create index %1', [1 => 'civicrm_membership_type.UI_name']), 'addIndex', 'civicrm_membership_type', [['name']], 'UI');
```

Comment:
@eileenmcnaughton I think somethings gone a bit wrong here. You've got duplicate upgrade tasks.

Reply:
@mattwire one task to add the fields then after the sql runs one to make them required